Makefile

```diff
@@ -95,7 +96,7 @@ endif
 AFLAGS = -fsanitize=address #-fsanitize=undefined -fno-omit-frame-pointer
 
 # Note: Intel oneAPI C/C++ compiler is now icx/icpx
-CC_VENDOR := $(subst icc_orig,icc,$(subst oneAPI,icc,$(firstword $(filter gcc clang icc icc_orig oneAPI XL emcc,$(subst -, ,$(shell $(CC) --version))))))
+CC_VENDOR := $(firstword $(filter gcc clang icc oneAPI XL emcc,$(subst -, ,$(shell $(CC) --version))))
```

Comment:
I'm not sure what to do about the bludgeoning of this line. The commit from the alcf-fork has a diff of:

```diff
-CC_VENDOR := $(subst icc_orig,icc,$(subst oneAPI,icc,$(firstword $(filter gcc clang icc icc_orig oneAPI XL emcc,$(subst -, ,$(shell $(CC) --version))))))
+CC_VENDOR := $(firstword $(filter gcc clang icc oneAPI XL emcc,$(subst -, ,$(shell $(CC) --version))))
```

which seemingly doesn't add anything? But I'm also not wise in the ways of `Make`, so I'm not sure if there was something actually wrong with the original line.

@kris-rowe do you remember why you made this change? (commit f9ab9a2a1 in the ALCF fork)

Reply:
It looks like we lost translation of `icc_orig`, which we've seen at some sites for their icc (classic, not icx) compilers. Can we restore that?

Does icx use clang-style defaults or is `-fp-model=fast` default? Is it okay to use clang(/gcc) options syntax (`-ffp-model`) instead of icc's?

Reply:
> It looks like we lost translation of `icc_orig`, which we've seen at some sites for their icc (classic, not icx) compilers. Can we restore that?

Yeah, that should be fixed now.

> Does icx use clang-style defaults or is `-fp-model=fast` default? Is it okay to use clang(/gcc) options syntax (`-ffp-model`) instead of icc's?

Not sure about that.